Utilities/Entities/Post.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Xml.Serialization;
+
+
+namespace QCUtilities.Entities
+{
+    [Serializable]
+    public class Post
+    {
+        [XmlElement(nameof(Author))]
+        public string Author { get; set; }
+
+        [XmlElement(nameof(Date))]
+        public DateTime Date { get; set; }
+
+        [XmlElement(nameof(Link))]
+        public string Link { get; set; }
+
+        [XmlElement(nameof(Title))]
+        public string Title { get; set; }
+
+        [XmlElement(nameof(Body))]
+        public string Body { get; set; }
+
+        public override bool Equals(object obj)
+        {
+            var item = obj as Post;
+            var result=
+            Author.Equals(item.Author)
+                &&
+                Date.Equals(item.Date)
+                &&
+                Link.Equals(item.Link)
+                &&
+                Title.Equals(item.Title)
+                &&
+                Body.Equals(item.Body)
+                ;
+            return result;
+        }
+
+        public override int GetHashCode()
+        {
+            return base.GetHashCode();
+        }
+    }
+
+    [Serializable]
+    [XmlRoot("Posts")]
+
```

Comment:
Another space between function attributes and function (I think I missed a few, also, go back and hunt for 'em)